app/controllers/users/registrations_controller.rb

```diff
@@ -8,7 +8,7 @@ def create_user_from_invitation
     @user.email = @invitation.email
     if @user.save
       sign_in @user
-      redirect_to accept_invitation_path(token: @invitation.token)
+      redirect_to accept_invitation_path(@invitation, token: @invitation.token)
```

Comment:
I did following in routes.rb
```  resources :invitations, only: [:index, :new, :create, :destroy] do
    member do
      get :accept
      get :resend
    end
  end
```
`accept_invitation_path	GET	/invitations/:id/accept(.:format)	invitations#accept`
shoud I change it to collection route?